src/components/TitleBar/DownloadBtn.tsx

```diff
@@ -19,20 +27,47 @@ export interface DownloadBtnProps {
  * @param {string} url
  * @param {string} filename
  */
-export const DownloadBtn = ({ url, filename }: DownloadBtnProps) => (
-  <a aria-label="Download" href={url} download={filename}>
-    <Download />
-    <style jsx>{`
-      * {
-        margin: 0;
-        padding: 0;
-      }
-      a {
-        color: ${Colors.sbGrayLighter};
-      }
-      a:hover {
-        color: ${Colors.sbGray};
+
+export class DownloadBtn extends Component<DownloadBtnProps, DownloadBtnState> {
+  constructor(props: DownloadBtnProps) {
+    super(props);
+    this.state = {
+      url: props.url,
+      filename: props.filename,
+      modal: {
+        taskModels: props.taskNames,
+        isOpen: false
       }
-    `}</style>
-  </a>
-);
+    };
+    // tslint:disable: no-unsafe-any no-any;
+    this.showHideModal = this.showHideModal.bind(this);
```

Comment:
Let's remove this and change `showHideModal` to an arrow function like we did earlier.